### ISSUE TITLE
Tab icon coloring

### ISSUE THREAD

**Issue by mrleblanc101** [file-icons/atom#652]
Hi,
I enabled `VCS Coloring` in `atom-tab`. Now my tab title are colored following their git status just like in `tree-view`.
The problem is the icons in the tab bar does not reflect the git status color like the one in the tab bar do.
Both icons in tree-view and atom-tab should reflect the same information, see screenshot for more details.
<img width="710" alt="capture d ecran 2017-09-30 a 22 16 39" src="https://user-images.githubusercontent.com/17392251/31051112-d7d649f0-a62d-11e7-8feb-efd11cb2011e.png">

**Comment by Alhadis** [comment]
I can't reproduce this... it appears to be working fine for me, and with every combination of relevant settings.

@mrleblanc101, did you observe this shortly after installing or reactivating the package? If so, a restart is needed before some behaviour can function properly. It's a known issue.

**Comment by mrleblanc101** [comment]
Restarted Atom, changed git repository, changed file type, changed git status (modified instead of new) and still not working
<img width="472" alt="capture d ecran 2017-10-01 a 16 23 01" src="https://user-images.githubusercontent.com/17392251/31058649-163d0318-a6c5-11e7-9c30-ade6fee82c76.png">

**Comment by Alhadis** [comment]
Alright, then we've got problems.

First, I'll need you to provide me with a full list of packages that you have installed. You can produce one by running the command `apm ls -lib` in a terminal, and copy+pasting its output. It'd also help if we knew which UI theme you're using, and what OS and Atom version you're running.

(I really need to write an issue template...)

**Comment by mrleblanc101** [comment]
Atom version: 1.20.1 (latest stable)
macOS version: 10.13.1 (beta 1)
Theme: Atom material
> atom-beautify@0.30.5
atom-clock@0.1.15
atom-ide-ui@0.4.0
atom-material-ui@2.0.4
atom-typescript@11.0.9
autoclose-html@0.23.0
docblockr@0.11.0
double-tag@0.9.0
emmet@2.4.3
file-icons@2.1.12
font-preview@0.1.1
highlight-selected@0.13.1
ide-typescript@0.1.6
language-antlers@0.3.4
minimap@4.29.7
minimap-cursorline@0.2.0
minimap-find-and-replace@4.5.2
minimap-git-diff@4.3.1
minimap-highlight-selected@4.6.1
minimap-pigments@0.2.2
minimap-selection@4.5.0
monokai@0.24.0
multiline-tab@0.2.0
multirow-tabs@0.3.3
php-twig@4.0.0
pigments@0.40.2
platformio-ide-terminal@2.7.0
svg-preview@0.11.0
svgo@2.8.5
sync-settings@0.8.3
tab-limiter@2.2.0
tab-preview@0.8.0
tag@0.5.0
tree-view-autoresize@2.2.0
tree-view-git-branch@0.1.2
tree-view-scope-lines@0.2.0

**Comment by mrleblanc101** [comment]
From the developer tools, I see that the problem comes from this CSS overwriting the .status-modified color.
```css
.amu-tinted-tab-bar atom-pane .tab-bar .tab .title[data-path]::before {
    /* color: rgba(255, 255, 255, 0.8) !important; */
}
<style>…</style>
atom-pane .tab-bar .tab .title[data-path]::before {
    margin-right: 16px;
    color: rgba(131, 148, 150, 0.65) !important;
}```

**Comment by mrleblanc101** [comment]
Seems like it comes from atom-material-ui/index.less

**Comment by Alhadis** [comment]
Looks like it's a known issue with the Atom Material theme. See [`atom-material/atom-material-ui#349`](https://github.com/atom-material/atom-material-ui/issues/349).

Guess I'd better send 'em a pull-request...

**Comment by Alhadis** [comment]
@mrleblanc101 Are you using a variant of the `atom-material` theme, or is there some setting for enabling a dark-coloured motif I'm not seeing? (Apologies for my ignorance, I don't use the Material themes).

**Comment by mrleblanc101** [comment]
I'm pretty sure the text editor color is set by the syntax theme, not the ui theme.
In that case, it would be solarized dark.

**Comment by Alhadis** [comment]
Ah, that would explain everything, since I use a light-coloured theme. Thanks!

**Comment by mrleblanc101** [comment]
When is the next release with the fix ?

**Comment by Alhadis** [comment]
Hey @mrleblanc101,

I'm actually waiting for the next release of [`atom-material-ui`](https://github.com/atom-material/atom-material-ui), which will make available the fix I submitted in [atom-material/atom-material-ui#439](https://github.com/atom-material/atom-material-ui/pull/439). The reason I've not cut a release yet is to catch any last-minute bug reports that might be submitted before there's a new theme release.

I apologise for not clarifying, I should've explained my inactivity a lot better. 😞

**Comment by Alhadis** [comment]
In other words, even if I make a new release now, you won't see the effects until another release of the Atom Material theme. You'll need to ask @silvestreh what his release cycle is like, I'm afraid.

**Comment by mrleblanc101** [comment]
Nah it's not your fault, it's alright.
I had an Atom Material update not long after I opened the issue so I thought the fix was already published.

**Comment by silvestreh** [comment]
I'll push an update for Material UI 😄 … My baby was born last week, so my head's been everywhere but Github!

**Comment by Alhadis** [comment]
HAH! Trust us, we understand. :D Family first, mate. And congratulations on the new bundle, so to speak. =D

**Comment by Alhadis** [comment]
Thanks for pushing a new release, mate! 😀

@mrleblanc101 I'll cut a release soon-ish, I'm still trying to fix some weirdness with the spec-runner (Atom's prompting me to save a file as it's running tests, not really cool...).

**Comment by Alhadis** [comment]
Published and released in [`v2.1.13`](https://github.com/file-icons/atom/releases/tag/v2.1.13), please enjoy. =)
